src/competitions/models.py

```diff
@@ -13,12 +13,28 @@ class Brand(models.Model):
     def __str__(self):
         return self.name
 
+# This is the categories catalog
+class Category(models.Model):
```

Comment:
It might be better to name the classes: BJCPCategory and BJCPStyle to have a more clear understanding on what they store.